src/main/views/macros/elements/task-list/types/subTask.ts

```diff
@@ -0,0 +1,13 @@
+import { TaskStatus } from "../enum/taskStatus";
```

Comment:
I am used to single quote and no semi-colon and i think some linters suggest that. I am open but let's be consistant.

Reply:
Why no semi-colon?